lib/transport/selenium-webdriver/service-builders/base-service.js

```diff
@@ -77,10 +57,6 @@ class BaseService {
     this.hostname = this.settings.webdriver.host || BaseService.DEFAULT_HOST;
     this.port = this.settings.webdriver.port;
 
-    if (!this.settings.webdriver.server_path && this.requiresDriverBinary) {
-      throw this.getStartupErrorMessage(this.errorMessages.binaryMissing);
-    }
```

Comment:
We can't remove this since both `Appium` and `Selenium` relies on this to throw erorr

Reply:
not removing.